lib/services/ios-project-service.ts

```diff
@@ -181,14 +181,15 @@ class IOSProjectService extends projectServiceBaseLib.PlatformProjectServiceBase
 			this.validateDynamicFramework(libraryPath).wait();
 			var umbrellaHeader = this.getUmbrellaHeaderFromDynamicFramework(libraryPath).wait();
 
-			var frameworkName = path.basename(libraryPath, path.extname(libraryPath));
-			var targetPath = path.join(this.$projectData.projectDir, "lib", this.platformData.normalizedPlatformName, frameworkName);
-			this.$fs.ensureDirectoryExists(targetPath).wait();
-			shell.cp("-R", libraryPath, targetPath);
+			let frameworkName = path.basename(libraryPath, path.extname(libraryPath));
+			let targetPath = path.join("lib", this.platformData.normalizedPlatformName, frameworkName);
+			let fullTargetPath = path.join(this.$projectData.projectDir, targetPath);
+			this.$fs.ensureDirectoryExists(fullTargetPath).wait();
+			shell.cp("-R", libraryPath, fullTargetPath);
 
 			let project = this.createPbxProj();
-
-			project.addFramework(path.join(targetPath, frameworkName + ".framework"), { customFramework: true, embed: true });
+			let frameworkPath = path.relative("platforms/ios", path.join(targetPath, frameworkName + ".framework"));
```

Comment:
platforms/ios - will this work on Windows? - This service is executed only on OSX.
Also, don't you need full paths on both places: - No, I don't need the fullPath, a just need the relative path to native project.